OmiseSDK/SourcePaymentMethod.swift

```diff
@@ -199,6 +199,37 @@ public enum PaymentInformation: Codable, Equatable {
     /// Mobile Banking Payment Source
     case mobileBanking(MobileBanking)
     
+    /// Internet Banking FPX
+    public struct FPX: PaymentMethod {
+        public static var paymentMethodTypePrefix: String = OMSSourceTypeValue.FPX.rawValue
+
+        public var type: String = OMSSourceTypeValue.FPX.rawValue
+
+        /// The customer's bank name
+        public let bank: String
```

Comment:
Do you think we could use an enum for the bank names? Would make it more convenient to use if you don't have to lookup the exact names.

Reply:
For this payment method. Bank names are dynamic list which contains 20+ banks.